### DISCUSSION TITLE
Como citar código externo

### DISCUSSION THREAD

**Discussion by AgustinSaavedra1** [IIC2133-PUC/2024-2#157] [Tarea 2] [answered]
Hola, me surgió la duda sobre como citar correctamente. Lo debo incluir como referencias en el informe o debo hacer algo mas?

**Comment by tomastrivino** [comment, marked as answer]
Te recomiendo incluirlo como referencia en el informe y además como un comentario dentro de tu código. Además, revisa el enunciado para entender cómo referenciar adecuadamente tu código.

Saludos
